src/harness.js

```diff
@@ -208,10 +208,11 @@ class Harness {
    * Loads and fills a contact form,
    *
    * @param {string} contactType Type of contact that should be created
+   * @param {string} action Either create or edit, defaults to create
    * @param  {...string[]} answers Provide an array for the answers given on each page. See fillForm for more details.
    */
-  async fillContactForm(contactType, ...answers) {
-    const xformFilePath = path.resolve(this.options.contactXFormFolderPath, `${contactType}-create.xml`);
+  async fillContactForm(contactType, action, ...answers) {
```

Comment:
this is a breaking change which will break all existing forms ... what other options can you think of?

Reply:
I think it'd be better if i created a `fillContactEditForm` or something alike because after testing this change it did not work as expected